src/wild_encounter.c

```diff
@@ -241,13 +268,19 @@ static u8 ChooseWildMonIndex_Fishing(u8 rod)
         else
             wildMonIndex = 1;
         break;
+
+        if (swap)
+            wildMonIndex = 1 - wildMonIndex;
```

Comment:
I guess I forgot to press the button to send this on my review, but shouldn't this `if (swap)` check be put *before* the `break;`?

Reply:
...yes.

Reply:
Yes it should. That's a mistake on my end.

Reply:
Fixed now.